src/from_geo.rs

```diff
@@ -35,21 +40,28 @@ impl<'a> TryInto<GGeom> for &'a LineString<f64> {
 
 struct LineRing<'a>(&'a LineString<f64>);
 
+/// Convert a geo::LineString to a geos LinearRing
+/// a LinearRing should be closed so cloase the geometry if needed
 impl<'a> TryInto<GGeom> for &'a LineRing<'a> {
     type Err = Error;
 
     fn try_into(self) -> Result<GGeom, Self::Err> {
         let points = &(self.0).0;
-        let coord_seq = create_coord_seq(&points)?;
+        let nb_points = points.len();
+        if nb_points >= 1 && nb_points < 3 {
```

Comment:
Wouldn't it be clearer to write `if nb_points < 3` ? And that would simplify the other conditions (lines 56,58).  
What is supposed to happen if nb_points <= 1 ?

Reply:
we want an empty `LinearRing` :cry: 

so I don't think we can cleanup this